src/shared/modules/markup.ts

```diff
@@ -0,0 +1,22 @@
+import isPlainObject from 'lodash/isPlainObject';
+
+import {MarkupItem} from '../types';
+
+export function isMarkupItem(obj: unknown): obj is MarkupItem {
+    return isPlainObject(obj) && isMarkupObject(obj as object);
+}
+
+function isMarkupObject(obj: {type?: string}): boolean {
+    if (
+        (obj.type === 'concat' ||
```

Comment:
Should we add all of [these](https://github.com/datalens-tech/datalens-ui/blob/main/src/ui/components/Markup/constants.ts#L1) types?